test/plugin/test_out_forward.rb

```diff
@@ -107,6 +107,28 @@ def test_configure_no_server
     end
   end
 
+  def test_configure_text_compress
```

Comment:
This test name doesn't claim anything. Does it show the default configuration value of `compress` option, right?